python/calculator.py

```diff
@@ -28,53 +29,90 @@ def calculate(self, order: int):
         elif order < 3:
             raise ValueError("order must be 3 or higher. Got {}".format(order))
         self.order = order
-        self.products = list()
+        
+        # pass in the first product so NUMBA can infer the types
+        # this is a hacky way to get around the fact that NUMBA doesn't support lists
+        # Remove it from the final result
```

Comment:
I have a feeling this hasn't been removed, has it?
When I run the original code, I get 1 fewer IMD product than running the numba code:

```
~/IMD-calculator/python$ python calculator.py 11 101 102 103 104 105 106 107 108 109 110  # original code
ORDER: 11
FREQUENCIES: [101.0, 102.0, 103.0, 104.0, 105.0, 106.0, 107.0, 108.0, 109.0, 110.0]
Time taken: 36.50073 seconds
Found 5188580 IMD products
```

```
~/IMD-calculator-1/python$ python calculator.py 11 101 102 103 104 105 106 107 108 109 110  # new code
... warnings ...
ORDER: 11
FREQUENCIES: [101.0, 102.0, 103.0, 104.0, 105.0, 106.0, 107.0, 108.0, 109.0, 110.0]
... warnings ...
Time taken: 5.24376 seconds
Found 5188581 IMD products
```

Reply:
you are right. I have pulled a version that removes the first item from the result.